api/src/main/java/io/opentelemetry/trace/samplers/DefaultSamplingDecision.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace.samplers;
+
+import io.opentelemetry.trace.SamplingDecision;
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * Sampling decision without tags.
+ *
+ * @since 0.1.0
+ */
+public class DefaultSamplingDecision implements SamplingDecision {
+
+  private final boolean decision;
+
+  /**
+   * Creates sampling decision without tags.
+   *
+   * @param decision sampling decision
+   * @since 0.1.0
+   */
+  public DefaultSamplingDecision(boolean decision) {
```

Comment:
Do we want to make it available in impls/SDK?

Reply:
Not necessary, we can always do that later.

Reply:
So I will change this to package protected for now